src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -346,12 +346,12 @@ export const wrapWorkspace = ({ breadcrumbs, activeTab, title, topBarContent, sh
       }
     }
 
-    const loadBucketLocation = async (googleProject, bucketName) => {
+    const loadBucketLocation = withErrorIgnoring(async (googleProject, bucketName) => {
```

Comment:
If you use use `withErrorIgnoring`, do you still need to do the async/await? This question came up in https://github.com/DataBiosphere/terra-ui/pull/3673#discussion_r1081877366

Reply:
WorkspaceContainer doesn't await `loadBucketLocation`, but I think awaiting the request in the wrapped function is necessary. Otherwise, I think a failed request would end up as unhandled promise rejection.

https://github.com/DataBiosphere/terra-ui/blob/57621d3ec31cd9e954da46491ac9d82cddac6850/src/libs/error.js#L15-L26

Reply:
That's my understanding as well… just wanted to make sure I was correct.